src/components/DelegationDashboard/DelegationTabs.tsx

```diff
@@ -0,0 +1,128 @@
+// Copyright 2019-2025 @polkassembly/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import classNames from 'classnames';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { Tabs } from '~src/ui-components/Tabs';
+import BecomeDelegate from './BecomeDelegate';
+import TotalDelegationData from './TotalDelegationData';
+import TrendingDelegates from './TrendingDelegates';
+import { Skeleton, TabsProps } from 'antd';
+import DelegationProfile from '~src/ui-components/DelegationProfile';
+import DashboardTrackListing from './TracksListing';
+import nextApiClientFetch from '~src/util/nextApiClientFetch';
+import { ProfileDetailsResponse } from '~src/auth/types';
+import { useUserDetailsSelector } from '~src/redux/selectors';
+
+interface Props {
+	className?: string;
+	theme?: string;
+	isLoggedOut: boolean;
+	userDetails: any;
+}
+
+const DelegationTabs = ({ className, theme, isLoggedOut, userDetails }: Props) => {
+	const userProfile = useUserDetailsSelector();
+	const { username, delegationDashboardAddress } = userProfile;
+	const [userBio, setUserBio] = useState<string>('');
+	const [profileDetails, setProfileDetails] = useState<ProfileDetailsResponse>({
+		addresses: [],
+		badges: [],
+		bio: '',
+		image: '',
+		social_links: [],
+		title: '',
+		user_id: 0,
+		username: ''
+	});
+	const [isModalOpen, setIsModalOpen] = useState(false);
+
+	const getData = async () => {
+		const { data, error } = await nextApiClientFetch(`api/v1/auth/data/userProfileWithUsername?username=${username}`);
+		if (data) {
+			setProfileDetails({ ...profileDetails, ...data });
+		} else {
+			console.log(error);
+		}
+	};
+
+	useEffect(() => {
+		getData();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [username, delegationDashboardAddress]);
+
+	const tabItems: TabsProps['items'] = [
+		{
+			children: (
+				<>
+					{isLoggedOut && <h2 className='mb-6 mt-5 text-2xl font-semibold text-bodyBlue dark:text-blue-dark-high max-lg:pt-[60px] md:mb-5'>Delegation </h2>}
```

Comment:
create a single component for children for both tabs (Dashboard and MyDelegation